### ISSUE TITLE
test code

### ISSUE THREAD

**Issue by geek-APTX4869** [cvlab-stonybrook/Beyond-Pixels#13]
Dear author：
  Can you provide the test code？

**Comment by prantikhowlader** [comment]
For Pascal (1/16,1/8,1/4) ResNet 101.
Please use the hyperparameters below 
Beta= 0.20(0-70 epochs), 0.12(70-100 epochs), after which 0.002
learning rate= 0.002
momentum=0.75
weight_decay=1e-3
scheduler= [factor=0.1, threshold=1e-3]
then use evaluate function in supervised.py to run the evaluation script
